src/components/select-project/selectProject.html

```diff
@@ -58,7 +58,8 @@ <h3 ng-if="$ctrl.canOnlyCreateProject()">Create Project</h3>
     ng-if="$ctrl.isNewProject()">
   <div class="form-group">
     <label for="name" class="control-label required">Project Name</label>
-    <div ng-class="{'has-error': ($ctrl.forms.createProjectForm.name.$error.pattern && $ctrl.forms.createProjectForm.name.$touched) || $ctrl.nameTaken}">
+    <div ng-class="{'has-error': ($ctrl.forms.createProjectForm.name.$error.pattern && $ctrl.forms.createProjectForm.name.$touched) ||
+                                 ($ctrl.nameTaken || $ctrl.forms.createProjectForm.name.$error.oscUnique)}">
```

Comment:
The parens around the last two conditions are unnecessary.

```html
 <div ng-class="{'has-error': ($ctrl.forms.createProjectForm.name.$error.pattern && $ctrl.forms.createProjectForm.name.$touched) ||
                               $ctrl.nameTaken || $ctrl.forms.createProjectForm.name.$error.oscUnique}">
```

Reply:
Fixed